modular-docs-manual/content/topics/assembly_creating_procedure_modules.adoc

```diff
@@ -0,0 +1,13 @@
+[[creating-procedure-modules]]
+= Creating Procedure Modules
+
+This section explains what a procedure module is and provides recommended practices for writing procedure modules.
+
+include::module_definition-procedure.adoc[leveloffset=+1]
+
+include::module_guidelines-procedure.adoc[leveloffset=+1]
+
+== Related Information
+
+* Download the link:https://gitlab.cee.redhat.com/ccs-internal-documentation/Modular_Documentation_Project/raw/master/files/TEMPLATE_PROCEDURE_doing_one_procedure.adoc[procedure module template] for new projects.
```

Comment:
This might get confusing if the actual template file is renamed later (which happened in the past, so I suppose it could happen again). For now, I changed the title name to this: [procedure module template (adoc file)]. Let me know if you have a better idea.